### DISCUSSION TITLE
skin folder where

### DISCUSSION THREAD

**Discussion by githubforzack** [ppy/osu#18572] [Q&A] [answered]
where is the skin folder in lazer i dug around in the files and i didnt find anything

mac btw

**Comment by frenzibyte** [comment, marked as answer]
User data is no longer publicly accessible, you'll have to export your skin, modify it, then import it back. We'll eventually add interfaces with the necessary features to modify skin sprites and configuration in-game.

You can read https://github.com/ppy/osu/wiki/User-file-storage for more information.
